specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1343,6 +1768,262 @@
+          "uniqueItems": true
+        },
+        "description": {
+          "type": "string",
+          "description": "A message passed to the owner of the remote NSP link resource with this connection request. Restricted to 140 chars."
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The NSP link state.",
+          "enum": [
+            "Approved",
+            "Pending",
+            "Rejected",
+            "Disconnected"
+          ],
+          "x-ms-enum": {
+            "name": "nspLinkStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Properties of NSP Link resource."
+    },
+    "NspLinkReconcile": {
+      "type": "object",
+      "properties": {},
+      "description": "Request object for NSP link reconcile."
+    },
+    "NspLinkReference": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NspLinkReferenceProperties",
+          "description": "Properties of the network security perimeter linkReference"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the linkReference resource that is unique within a perimeter. This name can be used to access the resource."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Identifier of the network security perimeter linkReference in ARM id format."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The network security perimeter linkReference resource"
+    },
+    "NspLinkReferenceListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NspLinkReference"
+          },
+          "description": "Gets a page of NSP LinkReference"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next page of results."
+        }
+      },
+      "description": "Result of the request to list NSP linkReference. Contains a list of NSP linkReference and a URL link to get the next set of results."
+    },
+    "NspLinkReferenceProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/NSPProvisioningState",
+          "description": "The provisioning state of the NSP LinkReference resource."
+        },
+        "remotePerimeterId": {
```

Comment:
>remotePerimeterId [](http://example.com/codeflow?start=9&length=17)

Can you please have this property name end with "resourceid" so it is clear that it is a arm resource id

Reply:
done.